src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
+    double area();
```

Comment:
```suggestion
    double getArea();
```

а вот названия методов должны начинаться с глаголов